style.css

```diff
@@ -0,0 +1,386 @@
+
+.see-more > p {
+  margin-bottom: 10px;
+}
+
+.see-more > p > i {
+  color: #ec5242;
+  padding-left: 4px;
+}
+
+.under {
+  text-decoration: underline;
+}
+
+.summit-sub {
+  background-color: #ec5242;
+  height: 40vh;
+  padding-top: 20%;
+}
+
+.summit-one {
+  background-image: url("assets/pastImg1.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 40vh;
+}
+
+.summit-two {
+  background-image: url("assets/pastImg2.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 40vh;
+}
+
+.more-partners {
+  background-image: url("assets/Headphones.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.first-group {
+  display: flex;
+  justify-content: space-around;
+}
+
+.first-p,
+.second-p,
+.third-p,
+.fourth-p,
+.fifth-p {
+  font-size: 30px;
+  font-weight: 700;
+  color: aquamarine;
+}
+
+.first-p {
+  color: #ec5242;
+}
+
+.third-p {
+  color: #ec5242;
+}
+
+.bottom-part {
+  display: flex;
+}
+
+.mobile-nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  transition: 0.2s ease-in-out;
+  background-color: #ec5242;
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  display: none;
+}
+
+.mobile-nav-ul > li > a {
+  color: #fff;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  text-decoration: none;
+}
+
+.nav-toggle {
+  display: flex;
+  flex-direction: column;
+}
+
+.mobile-nav-ul > li {
+  list-style-type: none;
+}
+
+.cross {
+  color: #fff;
+  font-size: 34px;
+  padding: 0 7px;
+  float: right;
+  margin-left: 85%;
+}
+
+.summit-sub:hover {
+  opacity: 0;
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 768px) {
+  #sponsor,
+  #updates {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .main-nav {
+    display: flex;
+    background-color: #3e3e3e;
+  }
+
+  .main-nav-ul {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 0;
+  }
+
+  .main-nav-i {
+    color: #fff;
+    list-style: none;
+    margin-left: 20px;
+  }
+
+  .main-nav-i:hover {
+    color: aquamarine;
+    list-style: none;
+    margin-left: 20px;
+    cursor: pointer;
+  }
+
+  #menu-image,
+  .see-more,
+  .about-salute,
+  .more-partners-about {
+    display: none;
+  }
+
+  .navigation {
+    display: contents;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .second-nav-ul {
+    display: flex;
+    margin-top: 25px;
+    justify-content: flex-end;
+  }
+
+  .second-nav-ul > li {
+    list-style: none;
+    padding-left: 30px;
+  }
+
+  .second-nav-i {
+    color: black;
+    text-decoration: none;
+  }
+
+  .second-nav-i:hover {
+    color: #ec5242;
+    text-decoration: none;
+  }
+
+  .hero-first-title {
+    font-size: 60px;
+    font-weight: 700;
+  }
+
+  .message-p {
+    width: 60%;
+    font-size: 20px;
+    font-weight: 600;
+  }
+
+  .program-container {
```

Comment:
In your program container, there are some changes to make

- [ ] Reducing the size of your font in this section will give you more space in the cards to work and will match also Behance design

- [ ] Kindly align the description of the text to match the rest of the cards

- [ ] Here there's a Lato font family too in the design

![image](https://github.com/Tarto-4/First-Capstone/assets/93630700/9acb9160-00bd-4e70-aab5-27cab717a225)

![image](https://github.com/Tarto-4/First-Capstone/assets/93630700/f3c72def-a140-470c-9ce6-1b5cf0e88ee7)